users_controller.py

```diff
@@ -0,0 +1,62 @@
+from flask import Flask, request, make_response
+from flask_restx import Api, Resource, fields
+
+app = Flask(__name__)
+api = Api(
+    app,
+    version="1.0",
+    title="barax",
+    description="REST API for our guild management application.",
+)
+
+
+@app.route("/users")
+def index():
+    all_users = {"users": users}
+    return all_users
+
+
+@app.route("/users", methods=["POST"])
+def create():
+    user = {
+        "name": request.json["name"],
+        "email": request.json["email"],
+        "password": request.json["password"],
+    }
+    users.append(user)
+    return make_response("", 204)
```

Comment:
We should look into a good way to validate input. Used regex in the past with a json model, not sure if that was good. Lets make an item in the backlog for it.